features/bevragen/protocollering.feature

```diff
@@ -0,0 +1,85 @@
+
+    Scenario: vragen om hele groep <groep>
+
+    Scenario: vragen om hele relatie <relatie>
+
+  Rule: Wanneer een gevraagd veld niet geleverd wordt, wordt het wel bij zoekrubrieken en niet bij geleverde rubrieken opgenomen
+    Een veld wordt niet geleverd wanneer deze geen waarde heeft, of boolean false, of wegens logica in de API niet geleverd moet worden
+
+      Voorbeelden:
+      | naam voorvoegsel heeft geen waarde         | naam                          |
+      | persoon heeft geen partner                 | partners.naam                 |
+      | persoon is niet overleden                  | overlijden.indicatieOverleden |
+      | verblijfstitel heeft einddatum in verleden | verblijfstitel                |
+      | niet leveren kind na ontkenning ouderschap | kinderen.naam.voornamen       |
+
+    Scenario: Vragen om aangaanHuwelijkPartnerschap bij een actuele partner
+    Scenario: Vragen om aangaanHuwelijkPartnerschap bij een ex-partner
+    Scenario: Vragen om ouders en persooon heeft ouder 1 en ouder 2
+    Scenario: Vragen om ouders en persoon heeft alleen ouder 1
+
+    Scenario: Wel opnemen als geleverd met standaardwaarde
+      | geboortedatum heeft alleen jaar bekend | leeftijd    |
+      | naam ouder is heeft standaardwaarde    | ouders.naam |
+
+  Rule: Wanneer een veld ongevraagd geleverd wordt, wordt het wel bij geleverde rubrieken en niet bij zoekrubrieken opgenomen
+    # To Do: vraag staat uit bij RvIG of dit waar is, dus of dit wel of juist niet geprotocolleerd moet worden
+    Dit betreft geheimhouding, opschorting, in onderzoek, RNI en verificatie
+
+  Rule: Wanneer een veld wordt gevraagd dat wordt bepaald uit een of meerdere andere velden, dan worden alle daarvoor nodige velden opgenomen als zoekrubrieken
+    En worden de feitelijk geleverde velden opgenomen als geleverde rubrieken
+
+      Voorbeelden:
+      | leeftijd                             |
+      | naam.voorletters                     |
+      | naam.volledigeNaam                   |
+      | overlijden.indicatieOverleden        |
+      | verblijfplaats.verblijfadres.straat  |
+      | verblijfplaats.datumVan              |
+      | adressering.aanschrijfwijze          |
+      | adressering.adresregel1              |
+      | adressering.adresregel2              |
+      | partners.aangaanHuwelijkPartnerschap |
+      | ouders                               |
+
+  Rule: Wanneer een veld wordt gevraagd waarbij meerdere velden worden meegeleverd voor het bepalen van het polymorfe type, dan worden alle daarvoor nodige velden opgenomen bij zoekrubrieken
+    En worden de feitelijk geleverde velden opgenomen als geleverde rubrieken
+
+      Voorbeelden:
+      | nationaliteiten.nationaliteit |
+      | nationaliteiten.redenOpname |
+      | verblijfplaats.functieAdres |
+  
+  Rule: Wanneer een gevraagd veld meerdere keren in het antwoord voorkomen, worden de geleverde rubrieken één keer opgenomen
```

Comment:
`Rule: Wanneer een gevraagd veld meerdere keren in het antwoord voorkomen, worden de geleverde rubrieken één keer in 'request_geleverde_rubrieken' vastgelegd`

Reply:
Rule: Wanneer een gevraagd veld meerdere keren in het antwoord voorkomt, worden de geleverde rubrieken één keer in 'request_geleverde_rubrieken' opgenomen.